content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,117 @@
+Nevertheless, every port to another architecture opens up possibilities for finding and fixing bugs in the kernel modules of ReactOS.
+
+Dmitry Borisov, a new ReactOS contributor, helped to get the PC-98 port in shape.
+While there is stuff left to be done in the development of PC-98 boot support, Dmitry Borisov made great strides to get ReactOS booting on that platform.
+The following screenshot below demonstrates ReactOS booting on Neko Project 21/W emulator.
+
+{{< figure link="/img/project-news/reactos-0414-released/nec98.png" src="/img/project-news/reactos-0414-released/nec98.png" >}}
+
+## ICMP improvements
+Internet Control Message Protocol (or ICMP by short) is a protocol used by network device drivers to send network information such as errors or log status.
+The implementation was rather scarce and incomplete with pieces of code unimplemented or unfinished.
+The IP Helper module primarily uses this protocol and as a matter of that, it brought a lot of issues and bugs. 
+
+Thanks to Victor Perevertkin, and with the help of Tim Crawford, the ICMP protocol implementation has seen lots of improvements.
+These range from the implementation of the IOCTL_ICMP_ECHO_REQUEST I/O control code to a full  rewrite of the Icmp** routines.
+As a result, the network device drivers can now handle ICMP requests in a correct manner.
+This is proven in the screenshot below, which shows the tracert command-line utility now outputting complete information.
+
+{{< figure link="/img/project-news/reactos-0414-released/icmp.png" src="/img/project-news/reactos-0414-released/icmp.png" >}}
+
+## Kernel improvements
+The NT kernel of ReactOS is the heart that is responsible for the well functioning of the system and communication with the bare metal.
+The stability of the system drastically relies on the robustness of the kernel.
+Every bug counts here.
+
+With every release, the developers squash bugs in the kernel to offer a better overall stability and experience.
+ReactOS 0.4.14 comes with several improvements to the Memory Manager thanks to Thomas Faber.
+Speaking of the aforementioned module, Vadim Galyant did some initial work on PAE (Physical Address Extension) and contributed towards the Memory Manager sections rewrite.
+
+Timo Kreuzer added CRT (C Run-Time) exception handling by importing related code from WINE. This allows 64-bit development to proceed as various bugs and random hangs on 64-bit systems have been resolved.
+Victor Perevertkin fixed a bug in storage class PnP driver (classpnp.sys) 
+and both Timo Kreuzer and Thomas Faber fixes some buffer overflows vulnerabilities. To finish with, Eric Kohl improved the device action worker code in I/O.
+
+Let's not forget the PnP (Plug n' Play) manager of the kernel which is also and important part not only in regard of stability the usability of the system to a plethora of hardware. Vadim Galyant added the initial headers implementation of the Resources Arbiter library which can be used by the bus
```

Comment:
Suggestion:
Let's not forget the PnP (Plug n' Play) manager of the kernel which is also an important...